ui/components/app/assets/token-list/token-list.tsx

```diff
@@ -101,16 +90,19 @@ export default function TokenList({
         filterCallback: 'inclusive',
       },
     ]);
-    const filteredSortedAssets = sortAssets(filteredAssets, tokenSortConfig);
-    return filteredSortedAssets;
+
+    const nativeTokens = filteredAssets.filter((token) => token.isNative);
+    const nonNativeTokens = filteredAssets.filter((token) => !token.isNative);
```

Comment:
If we use `.reduce` here we can avoid two loops.

Reply:
https://github.com/MetaMask/metamask-extension/pull/28193/commits/b541beb244827e5489073e1eb71aa2a604060132